cset-workflow/includes/deterministic_domain_mean_vertical_profile_series.cylc

```diff
@@ -1,6 +1,7 @@
 {% if DOMAIN_MEAN_VERTICAL_PROFILE_SERIES %}
 [runtime]
 {% for model_field in PRESSURE_LEVEL_MODEL_FIELDS %}
+{% for plevel in PRESSURE_LEVELS %}
```

Comment:
plevel isn't used. Should the PRESSURE_LEVELS in the options be plevel?

Reply:
good spot, I removed the for loop over pressure levels as not needed for plotting of vertical profiles. Git commit: 4a30a54..3413d4c